Formula/create-dmg.rb

```diff
@@ -0,0 +1,18 @@
+class CreateDmg < Formula
+  desc "Shell script to build fancy DMGs"
+  homepage "https://github.com/andreyvit/create-dmg"
+  url "https://github.com/andreyvit/create-dmg/archive/v1.0.0.4.tar.gz"
+  sha256 "4106eaccae7822db69870e6d36bad47d9d67e2fb690fbcb10ef488e3af061810"
+
+  def install
+    system "support/brew-me.sh"
+    bin.install "create-dmg"
+  end
+
+  test do
+    File.write(testpath/"Brew-Eula.txt", 'Eula')
```

Comment:
`  * C: 13: col 42: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.`

Reply:
Done!